Tests/BasicsTests/Environment/EnvironmentTests.swift

```diff
@@ -99,10 +102,13 @@ final class EnvironmentTests: XCTestCase {
 
     /// Important: This test is inherently race-prone, if it is proven to be
     /// flaky, it should run in a singled threaded environment/removed entirely.
-    func test_current() {
+    func test_current() throws {
+        try skipOnWindowsAsTestCurrentlyFails(because: "ProcessInfo.processInfo.environment[\"PATH\"] return nil")
```

Comment:
It should return `nil` as the variable is spelt `Path`.

Reply:
Thank you. The test has been updated to run and pass on Windows.